apps/nameguard.io/src/app/layout.tsx

```diff
@@ -39,6 +39,8 @@ export const metadata: Metadata = {
     ...NamehashMetadata.defaultMetdata.openGraph,
     title,
     description,
+    siteName: "NameGuard",
+    url: "https://nameguard.io",
```

Comment:
It would be better to use the VERCEL_PROJECT_PRODUCTION_URL environment variable here